common/web/gesture-recognizer/src/engine/eventSequentializationQueue.ts

```diff
@@ -0,0 +1,51 @@
+import { timedPromise } from "@keymanapp/web-utils";
+
+export class EventSequentializationQueue {
```

Comment:
A queue is sequential by design. The thing this is doing is dealing with async functions, not technically events. Maybe this should be called `AsyncFunctionQueue`.